src/Shared/Recruit.Vacancies.Client/Infrastructure/TableStore/TableStorageQueryStore.cs

```diff
@@ -18,8 +16,9 @@ namespace Esfa.Recruit.Vacancies.Client.Infrastructure.TableStore
 {
     internal sealed class TableStorageQueryStore : IQueryStore
     {
-        private readonly ILogger<TableStorageQueryStore> _logger;
+        private const int DeleteBatchSize = 100;
```

Comment:
Out of interest why was the delete failing against TableStorage when not batching?

Reply:
This wasn't failing (yet). Originally this retrieved all Live or Closed vacancies and then looped through them and deleted them one at a time.  When we have thousands of closed vacancies we won't be able to retrieve all of them at once.  By batching we only need to retrieve and process them 100 at a time.

Reply:
👍 good action